pkg/execution/svccatadmission/internaldns_test.go

```diff
@@ -44,30 +44,30 @@ func TestInternalDNSAdmitFunc(t *testing.T) {
 		{
 			"internaldns multiple new.domain",
 			buildAdmissionReview(dougComputeService, k8s.ServiceInstanceGVR, admissionv1beta1.Create, buildServiceInstance(
-				t, apiinternaldns.ClusterServiceClassExternalID, apiinternaldns.ClusterServicePlanExternalID, apiinternaldns.Spec{
-					Aliases: []apiinternaldns.Alias{
+				t, apiplatformdns.ClusterServiceClassExternalID, apiplatformdns.ClusterServicePlanExternalID, apiplatformdns.Spec{
+					Aliases: []apiplatformdns.Alias{
 						{
-							AliasType: apiinternaldns.AliasTypeSimple,
+							AliasType: apiplatformdns.AliasTypeSimple,
 							Name:      "new1.domain",
```

Comment:
Not part of this PR, but I'm wondering if these test domains shouldn't be restricted to
new1.atl-paas.net
Since I expect there will be various rules later that check that the alias is a subdomain of atl-paas.net.
Or we can keep it until the tests break

Reply:
I like the idea of waiting the tests to break.

Reply:
Right now, behaviour will be a failure on the provider side since a non-atl-paas domain will not be in the list of valid zones during provisioning.

Reply:
nit: not in provider side, on smith service catalog schema validation (before it actually makes the call to the provider)